src/main/kotlin/string_calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,16 @@
+package string_calculator
+
+class StringAddCalculator {
```

Comment:
객체가 상태 값을 지니지 않기에, object 키워드를 사용해 싱글턴 객체로 만드는 것도 좋아 보입니다 :)
```suggestion

object StringAddCalculator {
```